lib/src/command/logout.dart

```diff
@@ -16,12 +16,16 @@ class LogoutCommand extends PubCommand {
   @override
   String get description => 'Log out of pub.dev.';
   @override
-  bool get takesArguments => false;
+  bool get takesArguments => true;
 
   LogoutCommand();
 
   @override
   Future<void> runProtected() async {
-    oauth2.logout(cache);
+    if (argResults.rest.isEmpty) {
+      oauth2.logout(cache);
+    } else {
```

Comment:
You can't just take arguments and then throw them away... If there is multiple arguments, but you only accept one then we have to throw a usage error..

Reply:
Actually I thought about that, but then forgot to write it 😅 Anyways done.